src/cascadia/TerminalControl/UiaTextRange.cpp

```diff
@@ -24,12 +24,11 @@ HRESULT UiaTextRange::GetSelectionRanges(_In_ IUiaData* pData,
         // create a range for each row
         for (const auto& rect : rectangles)
         {
-            ScreenInfoRow currentRow = rect.Top();
-            Endpoint start = _screenInfoRowToEndpoint(pData, currentRow) + rect.Left();
-            Endpoint end = _screenInfoRowToEndpoint(pData, currentRow) + rect.RightInclusive();
+            const auto start = rect.Origin();
+            const auto end = rect.EndInclusive();

```

Comment:
I'm concerned- End is supposed to be _exclusive_ in the new UiaTextRange, right?

Reply:
https://github.com/microsoft/terminal/pull/4018/files#r372639804